framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -195,9 +196,34 @@ private void viewpointAdaptExecutable(
             receiver = FlowExpressions.internalReprOf(factory, receiverTree);
         }
 
-        List<FlowExpressions.Receiver> argReceivers = new ArrayList<>(args.size());
-        for (ExpressionTree argTree : args) {
-            argReceivers.add(FlowExpressions.internalReprOf(factory, argTree));
+        List<FlowExpressions.Receiver> argReceivers = new ArrayList<>();
+        boolean varargs = false;
+        if (tree instanceof MethodInvocationTree) {
+            ExpressionTree methodElement = ((MethodInvocationTree) tree).getMethodSelect();
+            ExecutableElement methodCalled =
+                    (ExecutableElement) TreeUtils.elementFromUse(methodElement);
```

Comment:
You should call `elementFormUse` on the method invocation tree, not the method select.  `elementFormUse((MethodInvocationTree) tree)`